deploy/manager/dev/10_scylla_cluster.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: scylla.scylladb.com/v1
+kind: ScyllaCluster
+metadata:
+  name: scylla-manager-cluster
+  namespace: scylla-manager
+spec:
+  agentVersion: 2.2.0
+  datacenter:
+    name: manager-dc
+    racks:
+      - members: 1
```

Comment:
nit: indentation

Reply:
This indendation is generated by the `yq`, unfortunately they decided to stay with additional indentation for arrays: https://github.com/mikefarah/yq/issues/194